sycl/test/check_device_code/ap_fixed.cpp

```diff
@@ -8,103 +8,59 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "CL/__spirv/spirv_ops.hpp"
+#include <sycl/sycl.hpp>
 
-template <int W, int rW, bool S, int I, int rI>
-void sqrt() {
-  sycl::detail::ap_int<W> a;
-  auto ap_fixed_Sqrt = __spirv_FixedSqrtINTEL<W, rW>(a, S, I, rI);
+SYCL_EXTERNAL void test_sqrt(sycl::detail::ap_int<13> a) {
+  auto ap_fixed_Sqrt = __spirv_FixedSqrtINTEL<13, 5>(a, false, 2, 2);
```

Comment:
```suggestion
SYCL_EXTERNAL auto test_sqrt(sycl::detail::ap_int<13> a) {
  return __spirv_FixedSqrtINTEL<13, 5>(a, false, 2, 2);
```

And I think that would resolve my question above as well.

Reply:
Fixed, thanks for the pointer!